tests/test_dc_devices.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import pytest
+from ophyd.sim import make_fake_device
+from pcdsdevices.dc_devices import ICT
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def fake_ICT():
+    FakeICT = make_fake_device(ICT)
+    PDU = FakeICT('TST:PDU:ICT', name='test')
+    PDU.ch1A.ch_status.sim_set_enum_strs(['ENABLED', 'DISABLED'])
+    PDU.ch1A.ch_status.sim_put('ENABLED')
```

Comment:
The automated tests are failing at setup because in the class you use `ch_1A` but in the tests you use `ch1A`.